plugins/module_utils/foreman_helper.py

```diff
@@ -257,10 +258,11 @@ def __init__(self, argument_spec, **kwargs):
         args.update(argument_spec)
         super(ForemanEntityAnsibleModule, self).__init__(argument_spec=args, **kwargs)
 
+        self.state = self.params.pop('state')
+        self.absent = self.state == 'absent'
```

Comment:
```suggestion
        self.desired_absent = self.state == 'absent'
```

Reply:
How could that escape my sight.

Reply:
Right. Those reduce your field of view...